docker-images/kafka-based/kafka/scripts/kafka_run.sh

```diff
@@ -65,27 +65,30 @@ echo "Using KRaft [${USE_KRAFT}]"
 
 # Prepare for Kraft
 if [ "$USE_KRAFT" == "true" ]; then
-  KRAFT_LOG_DIR=$(grep "log\.dirs=" /tmp/strimzi.properties | sed "s/log\.dirs=*//")
-
-  if [ ! -f "$KRAFT_LOG_DIR/meta.properties" ]; then
-    STRIMZI_CLUSTER_ID=$(cat "$KAFKA_HOME/custom-config/cluster.id")
-    METADATA_VERSION=$(cat "$KAFKA_HOME/custom-config/metadata.version")
-    echo "Formatting Kraft storage with cluster ID $STRIMZI_CLUSTER_ID and metadata version $METADATA_VERSION"
-    mkdir -p "$KRAFT_LOG_DIR"
-    # Using "=" to assign arguments for the Kafka storage tool to avoid issues if the generated
-    # cluster ID starts with a "-". See https://issues.apache.org/jira/browse/KAFKA-15754
-    ./bin/kafka-storage.sh format -t="$STRIMZI_CLUSTER_ID" -r="$METADATA_VERSION" -c=/tmp/strimzi.properties
+  # Format the KRaft storage
+  STRIMZI_CLUSTER_ID=$(cat "$KAFKA_HOME/custom-config/cluster.id")
+  METADATA_VERSION=$(cat "$KAFKA_HOME/custom-config/metadata.version")
+  echo "Making sure the Kraft storage is formatted with cluster ID $STRIMZI_CLUSTER_ID and metadata version $METADATA_VERSION"
+  # Using "=" to assign arguments for the Kafka storage tool to avoid issues if the generated
+  # cluster ID starts with a "-". See https://issues.apache.org/jira/browse/KAFKA-15754
+  ./bin/kafka-storage.sh format -t="$STRIMZI_CLUSTER_ID" -r="$METADATA_VERSION" -c=/tmp/strimzi.properties -g
```

Comment:
wondering if it's worth mentioning that the `-g` option allows to skip if the storage is already formatted.